dependencies/FindFastJet.cmake

```diff
@@ -46,43 +46,47 @@ if(${PKGNAME}_CONFIG)
                   ERROR_VARIABLE error
                   OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  set(${PKGNAME}_LIBS)
-  set(${PKGNAME}_LIB_DIRS)
-  mark_as_advanced(${PKGNAME}_LIBS ${PKGNAME}_LIB_DIRS)
-  if(${PKGNAME}_CONFIGLIBS)
-    string(REGEX MATCHALL "(^| )-l[^ ]+" libs ${${PKGNAME}_CONFIGLIBS})
-    foreach(lib ${libs})
-        string(STRIP ${lib} lib)
-        string(SUBSTRING ${lib} 2 -1 lib)
-        list(APPEND ${PKGNAME}_LIBS ${lib})
-    endforeach()
-
-    string(REGEX MATCHALL "(^| )-L[^ ]+" libdirs ${${PKGNAME}_CONFIGLIBS})
-    foreach(libdir ${libdirs})
-        string(STRIP ${libdir} libdir)
-        string(SUBSTRING ${libdir} 2 -1 libdir)
-        list(APPEND ${PKGNAME}_LIB_DIRS ${libdir})
-    endforeach()
-
-    # append directories of exposed libraries
-    if ("CGAL" IN_LIST ${PKGNAME}_LIBS)
-      find_library(lib_cgal NAMES "CGAL" PATHS $ENV{CGAL_ROOT}/lib NO_DEFAULT_PATH)
-      if (NOT lib_cgal)
-        message(FATAL_ERROR "CGAL not found in $ENV{CGAL_ROOT}/lib")
+  if (${PKGNAME}_FOUND})
```

Comment:
I fail to see how this is supposed to work, `${PKGNAME}_FOUND` will only be set by the macro in line 93 (in your version). What do I miss?

In fact, I don't see the need for any protection here.

Reply:
you need to hide whitespace changes, this protection is actually at a different place as you might think when looking at the full diff. `${PKGNAME}_FOUND` is set by the configuration command, and only i the configuration command finds the package, the additional checks should be executed.

Reply:
@matthiasrichter Which "configuration command" are you talking about? Lacking a FastJetConfig.cmake, we fall back to `fastjet-config`, but this does not set any cmake variables.

PS: Yes, I looked at the full file (and with whitespace changes hidden).

Reply:
Correct, I'm on the wrong path.